File: MusicManagerAPI/src/main/java/com/ldz/music/manager/repository/UserMusicStatusRepository.java
package com.ldz.music.manager.repository;

import com.ldz.music.manager.model.UserMusicStatus;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * Created by ldalzotto on 26/04/2017.
 */
public interface UserMusicStatusRepository extends JpaRepository<UserMusicStatus, Long> {

    public void deleteByUsername(String username);

    public UserMusicStatus findByUsername(String username);

}

File: MusicManagerAPI/src/scala/com/ldz/music/manager/repository/UserMusicStatusRepository.scala
package com.ldz.music.manager.repository

import com.ldz.music.manager.model.UserMusicStatus
import org.springframework.data.jpa.repository.JpaRepository

/**
  * Created by Loic on 20/06/2017.
  */
trait UserMusicStatusRepository extends JpaRepository[UserMusicStatus, java.lang.Long]{

  def deleteByUsername(username: String): Unit
  def findByUsername(username: String): UserMusicStatus
}
